run.py

```diff
@@ -1,29 +1,34 @@
 #!/usr/bin/env python3
 
+import argparse
 import os
-import sys
-import time
+import pathlib
 import shutil
 import signal
+import sys
+import time
+from threading import Thread
+from typing import List
+
 import pexpect
-import pathlib
-import argparse
 import termcolor
 
+from ebpf.ebpf import attach_ebpf, attach_probes, harvest_ebpf
```

Comment:
What if eBPF is not installed? Do we make it a core dependency? How is it being pulled?

Reply:
Changed this to not require BCC toolkit, if `--with-ebpf` flag is absent. Also added instructions link to `Readme.md`.